test-integration/helpers/reading-builder.coffee

```diff
@@ -269,10 +273,11 @@ class ReadingBuilder extends TestHelper
           @cancel() if isDisplayed
       else
         # Click "Add Readings"
+        @test.sleep(100) # Not sure why this is needed
         @el.addReadingsButton.click()
```

Comment:
My pref would be to add timeouts _very sparingly_ precisely because it introduces transient errors, no?